.storybook/preview.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect } from 'react'
+import { ChakraProvider, useColorMode } from '@chakra-ui/react'
+import { Story } from '@storybook/react'
+import { customTheme } from '../app/theme'
+
+type ColorModeProps = {
+  colorMode: 'light' | 'dark'
+  children: JSX.Element
+}
+
+function ColorMode(props: ColorModeProps) {
+  const { setColorMode } = useColorMode()
+
+  useEffect(() => {
+    setColorMode(props.colorMode)
+  }, [props.colorMode])
+
+  return props.children
+}
+
+const withChakra = (Story: Story, context) => {
```

Comment:
I recommend using StoryFn here instead of Story, as Story is now deprecated :raising_hand:

<img width="551" alt="Screenshot 2023-11-22 at 14 55 24" src="https://github.com/tabi-memo/frontend/assets/7115171/d0f77b87-d567-4ec4-a5aa-ee5a06b127aa">

Reply:
Thank you, You're right! I changed it
https://github.com/tabi-memo/frontend/pull/2/commits/da22299fe6cd295fd1085bd39705bfaf566fe6f9